src/redux/Books/Books.js

```diff
@@ -0,0 +1,26 @@
+// actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// reducer
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return 'add a book';
+    case REMOVE_BOOK:
+      return 'remove a book';
```

Comment:
- [ ] Same issue here, you  should filter out the book you want to remove from your state and not simply return `remove a book`